### ISSUE TITLE
[EN][NPC][Dragonblight] Indu'le Warrior & Fisherman

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4197]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Alot of times these NPC's do not drop anything at all.

**Expected behaviour**: Tell us what should happen instead
Being human nature but are undead they still have pockets and should carry currency and items on loot.
Each one should have a currency value of roughly 14silver+

**Steps to reproduce the problem**:
1. Kill a few Indu'le see if lootabale
2.
3. 

**Include proofs for this behaviour**
You can see in the chat log of the xp gained and what was or may not be looted.
![wowscrnshot_011117_210706](https://cloud.githubusercontent.com/assets/12252441/21874363/97b76d5e-d842-11e6-8b0d-3dbf7d3b90e5.jpg)

![wowscrnshot_011117_210740](https://cloud.githubusercontent.com/assets/12252441/21874366/9da6c4c6-d842-11e6-9d26-a24780b83cd8.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Open Wow Links
[Indu'le Warrior](http://wotlk.openwow.com/npc=26344)
[Indu'le Fisherman](http://wotlk.openwow.com/npc=26343)

Wowhead Links
[Warrior](http://www.wowhead.com/npc=26344/indule-warrior)
[Fisherman](http://www.wowhead.com/npc=26343/indule-fisherman)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Creature: Indu'le Warrior & Fisherman - Gold + Loot

* both will now drop around 10 - 20 silver and always contain at least one trash item
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4197
* commit: 88e7336351483225219128948ca5c22a55d9f96f

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
